nautobot/docs/additional-features/caching.md

```diff
@@ -64,3 +64,115 @@ Finally, calling it with the `all` argument will force invalidation of the entir
+of the Redis server and port for each sentinel instance to connect to
+* `LOCATION`: Similar to a redis URL, *however*, the hostname in the URL is the master/service name in redis sentinel
+* `SENTINEL_KWARGS`: Options which will be passed directly to [Redis Sentinel](https://github.com/redis/redis-py#sentinel-support)
+* `PASSWORD`: The redis password (if set), the `SENTINEL_KWARGS["password"]` setting is the password for Sentinel
+
+Example:
+
+```python
+DJANGO_REDIS_CONNECTION_FACTORY = "django_redis.pool.SentinelConnectionFactory"
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": "redis://nautobot/0",  # in this context 'nautobot' is the redis master/service name
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.SentinelClient",
+            "CONNECTION_POOL_CLASS": "redis.sentinel.SentinelConnectionPool",
+            "PASSWORD": "",
+            "SENTINEL_KWARGS": {
+                "password": "",  # likely the same password from above
+            },
+            "SENTINELS": [
+                ("mysentinel.redis.example.com", 26379),
+                ("othersentinel.redis.example.com", 26379)
+            ],
+        },
+    },
+}
+```
+
+!!! note
+    It is permissible to use Sentinel for only one database and not the other, see [`RQ_QUEUES`](../../configuration/required-settings/#rq_queues) for details.
+
+For more details on configuring django-redis with Redis Sentinel, please see the documentation for [Django Redis](https://github.com/jazzband/django-redis#use-the-sentinel-connection-factory).
+
+### `django-cacheops` Sentinel Configuration
+
+Notable settings:
+
+* `locations`: List of tuples or tuple of tuples with each inner tuple containing the name or IP address
+of the Redis server and port for each sentinel instance to connect to
+* `service_name`: the master/service name in redis sentinel
+* Additional parameters may be specified in the `CACHEOPS_SENTINEL` dictionary which are passed directly to Sentinel
+
+!!! note
+    `locations` for `django-cacheops` has a different meaning than the `LOCATION` value for `django-redis`
+
+!!! warning
+    [`CACHEOPS_REDIS`](#cacheops_redis) and [`CACHEOPS_SENTINEL`](#cacheops_sentinel) are mutually exclusive and will result in an error if both are set.
+
+Example:
+
+```python
+CACHEOPS_REDIS = False
+CACHEOPS_SENTINEL = {
+    "db": 0,
+    "locations": [
+        ("mysentinel.redis.example.com", 26379),
+        ("othersentinel.redis.example.com", 26379)
+    ],
+    "service_name": "nautobot",
+    "socket_timeout": 10,
+    "sentinel_kwargs": {
+        "password": ""
+    },
+    "password": "",
+    # Everything else is passed to `Sentinel()`
+}
+```
+
+For more details on how to configure Cacheops to use Redis Sentinel see the documentation for [Cacheops setup](https://github.com/Suor/django-cacheops#setup).
+
+### `celery` Sentinel Configuration
```

Comment:
Technically this section isn't about caching so i wouldn't expect it in the caching section of the doc.
I understand it's a hard one and I'm not sure if there is a better place but I think we should be careful to position Redis just for caching